### PR TITLE
Add method to get instance for Altspace space

### PR DESCRIPTION
**Pull request by amberroy**

### CODE REVIEW

File: src/utilities/sync.js [hunk 1]
```diff
@@ -54,6 +54,49 @@ altspace.utilities.sync = (function() {
         return firebaseInstance;
     }
 
+    function getInstanceForSpace(params, callback) {//TODO: document
```
Comment (brianpeiris) [inline src/utilities/sync.js]: Return a promise instead of using a callback.

File: src/utilities/sync.js [hunk 2]
```diff
@@ -54,6 +54,49 @@ altspace.utilities.sync = (function() {
...
+        roomId = spaceSid;
+        console.log("No roomId in URL, using Altspace space sid: " + roomId);
+      } else {
+        roomId = params.defaultRoomId || 'default';
+        console.log("No roomId in URL and not running in Altspace, using default room: " + roomId);
+      }
+
+      // Connect to Firebase
+      params["instanceId"] = roomId;
+      return getInstance(params, spaceSid);
```
Comment (brianpeiris) [inline src/utilities/sync.js]: getInstance doesn't take a second parameter.

File: src/utilities/sync.js [hunk 3]
```diff
@@ -54,6 +54,49 @@ altspace.utilities.sync = (function() {
...
+      } else {
+        instance = connect(params);
+        if (callback) callback(instance);
+      }      
+    }
+
+    function connect(params, spaceSid) {//TODO: document
+
+      //three cases: roomId in the URL, use Altspace sid, or use default roomId
+      var roomId = url.query['roomId'];
```
Comment (brianpeiris) [inline src/utilities/sync.js]: `url` is not defined. Er, I guess this PR is incomplete.

File: src/utilities/sync.js (hunk 1 above)
Comment (amberroy) [inline src/utilities/sync.js]: ah yes you're right that would be more consistent with our other methods.

File: src/utilities/sync.js (hunk 3 above)
Comment (amberroy) [inline src/utilities/sync.js]: thanks for the review, I'll fix it up.

**Pull request closed without merging**